core/common-ui/src/main/java/team/noweekend/core/common/ui/calendar/component/CalendarDay.kt

```diff
@@ -0,0 +1,107 @@
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.graphicsLayer
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import team.noweekend.core.common.ui.calendar.model.DateOfWeek
+import team.noweekend.core.common.ui.calendar.state.CalendarPagerState
+import team.noweekend.core.common.ui.calendar.state.CalendarPagerState.CalendarMode
+import java.time.LocalDate
+
+@Composable
+internal fun CalendarDay(
+    dateOfWeek: DateOfWeek,
+    calendarMode: CalendarMode,
+    modifier: Modifier = Modifier,
+    isSelectedDay: Boolean = false,
+    isCurrentMonth: Boolean = false,
+    horizontalAlignment: Alignment.Horizontal = Alignment.CenterHorizontally,
+    onClickDateOfWeek: (DateOfWeek) -> Unit = {},
+) {
+    val condition = if (calendarMode == CalendarMode.MONTH) {
+        if (isCurrentMonth) 1f else 0f
+    } else {
+        1f
+    }
+    Column(
+        modifier = modifier
+            .graphicsLayer {
+                alpha = condition
+            }
+            .clickable(
+                enabled = isCurrentMonth,
+            ) {
+                onClickDateOfWeek(dateOfWeek)
+            },
+        horizontalAlignment = horizontalAlignment,
+    ) {
+        Row(
+            modifier = Modifier
+                .size(41.dp)
+                .padding(4.5.dp)
+                .background(
+                    if (isSelectedDay) Color(0xFFFFEAE0) else Color.Transparent,
+                    shape = CircleShape,
+                ),
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.Center,
+        ) {
+            Text(
+                text = dateOfWeek.localDate.dayOfMonth.toString(),
+                color = if (isSelectedDay) {
+                    Color(0xFFD64000)
+                } else {
+                    Color(0xFF333333)
+                },
+                textAlign = TextAlign.Center,
+            )
+        }
+        Row(
+            modifier = Modifier.size(41.dp),
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.Center,
+        ) {
+            Image(
+                modifier = Modifier.fillMaxSize(),
+                painter = painterResource(id = dateOfWeek.imageType.id),
+                contentDescription = null,
+            )
+        }
```

Comment:
```suggestion
        Image(
            modifier = Modifier.size(41.dp),
            painter = painterResource(id = dateOfWeek.imageType.id),
            contentDescription = null,
        )
```

p3 : 위에 Text처럼 백그라운드에 별도의 테마가 들어가는게 아닌 것 같아서 그냥 size modifier만 추가해도 괜찮을 것 같은데 특별한 이유가 있을까용??
그리고 이거 이미지 visibility는 아직 안들어간거죵?

Reply:
@014967 요고 체크 못했던거같은데??? 한번 확인해줭

Reply:
[bbd4030](https://github.com/mash-up-kr/NoWeekend-Android/pull/32/commits/bbd403043ff4cca980da8ffa9e9efd54c1e8e2c1)
반완